crates/fuzzing/src/oracles.rs

```diff
@@ -418,7 +418,8 @@ pub fn table_ops(config: crate::generators::Config, ops: crate::generators::tabl
         let engine = Engine::new(&config);
         let store = Store::new(&engine);
 
-        let wat = ops.to_wat_string();
+        let wat = ops.to_wasm_binary();
+        let wat = wasmprinter::print_bytes(&wat).unwrap();
         log_wat(&wat);
```

Comment:
This will let us avoid the wasm->wat disassembly (and thus have higher fuzzing throughput) unless logging is enabled:

```suggestion
        let wasm = ops.to_wasm_binary();
        log_wasm(&wasm);
```